otterdog/eclipse-ecal.jsonnet

```
@@ -217,6 +217,7 @@ orgs.newOrg('automotive.ecal', 'eclipse-ecal') {
       allow_merge_commit: true,
       allow_update_branch: false,
       delete_branch_on_merge: false,
+      private_vulnerability_reporting_enabled = true,
```

must be a colon `:`, not an equal `=`
```suggestion
      private_vulnerability_reporting_enabled: true,
```

---

Oh yes, that was the issue! Thank you.